cartography/intel/github/util.py

```diff
@@ -88,11 +89,13 @@ def fetch_all(
             )
             raise
         except requests.exceptions.HTTPError:
-            logger.error(
-                f"GitHub: Could not retrieve page of resource `{resource_type}` due to HTTP error.",
-                exc_info=True,
-            )
-            raise
+            retry += 1
+            if retry <= retries:
+                logger.error(
+                    f"GitHub: Could not retrieve page of resource `{resource_type}` due to HTTP error.",
+                    exc_info=True,
+                )
+                raise
```

Comment:
I _think_ we need a `continue` here otherwise it will continue on to line 99 which will result in a KeyError because `resp` will be falsy.

```suggestion
                raise
            else:
                continue
```